### ISSUE TITLE
Wave of bugs - Mask of the Betrayer

### ISSUE THREAD

**Issue by Felipefpl** [nwn2fixes/player#53]
Now that i finished OC (for the 1st time) i'm gonna put the bugs i find here for future reference :) :

The camera is too close to the character, i know i reported this in OC but this happened only when i started the game, now it happens most of times when i load a savegame inside mulsantir, the bad behaviour of OC remained while i was in the barrow/outside the city. After i went inside it got worse.

Vyk at ice lodge is bald according to description but his pic shows a man with hair.

I helped Yulia at ice troll lodge defeating Forovan at the arm's contest but no xp was given and no entry in the journal was added, dev's intention or a bug?

Helped vyk and radu in the ice lodge to train and again no xp was given/no journal's entry added, a bug or was intended by the devs?

I raised using the points on level up the skills Diplomacy = 15 and intimidate = 26 (the max i could raise them at OC) and even so no success with gann's conversation at the prison, neither talking with sheva about okku, the values should be bigger or is it a bug?

If i do a quick save during the fight with Jurak to be accepted in the lodge and load that quick save he'll wont be in the "ring" anymore, instead he'll be outside attacking my part members, savegame attached.

[Jurak.zip](https://github.com/nwn2fixes/player/files/6572402/Jurak.zip)

Now, during the fight against sergey if i load a quick save inside the "ring" when i'm fighting him Safyia will shoot arrows at him.

Also, during all the tests i go through at the lodge no entry in the journal is added, is it by design or a bug?

I get xp for lockpicking the doors at the Sloop inn but no xp when i do the same to chests or when i disable/recover traps, if this happens by devs' decision it was a bad one and it makes no sense.

The last letter of influence and the number associated with it in the character screen overlap.

**Comment by kevL** [comment]
heya Felipe,
it's kinda neat that you're adding all this stuff. (it's nonstandard use of Issues, but im okay with it :)

| The game saved by itself several times when i'm on shadow mulsantir, this happened after i finish loading the game.
probly the scripted autosave -- the designers put autosaves in every so often ... agreed it should be an option but it doesn't seem to break anything

**Comment by Felipefpl** [comment]
> heya Felipe,
> it's kinda neat that you're adding all this stuff. (it's nonstandard use of Issues, but im okay with it :)

Nice, but i didnt know it was non-standard though, good you are fine with that.
 
> | The game saved by itself several times when i'm on shadow mulsantir, this happened after i finish loading the game.
> probly the scripted autosave -- the designers put autosaves in every so often ... agreed it should be an option but it doesn't seem to break anything

Doesnt autosave works on specific places like the waypoints from one scenery to other one? In these specific cases the game is being saved several times after i load in shadow mulsantir but since you said it's by design i'd say it was a bad decision from the devs. :/

**Comment by kevL** [comment]
| i didnt know it was non-standard though

just to let you know, in case you're active on other git repos, others often won't like a bunch of issues like this all posted into one Issue -- usually they should each be entered as a unique bug/feature :)

| In these specific cases the game is being saved several times after i load in shadow mulsantir but since you said it's by design i'd say it was a bad decision from the devs.

yeh they made a lot of 'hasty' decisions. Autosave is a function that is typically called by an OnEnter script -- when the PC enters an area, the OnEnter script runs the autosave function -- it should be guarded by a condition so it doesn't happen more than once. Apparently someone left out the condition ...

**Comment by Felipefpl** [comment]
> just to let you know, in case you're active on other git repos, others often won't like a bunch of issues like this all posted into one Issue -- usually they should each be entered as a unique bug/feature :)

I see, i dont understand why, imagine a github full of the issues i post here. :P

> yeh they made a lot of 'hasty' decisions. Autosave is a function that is typically called by an OnEnter script -- when the PC enters an area, the OnEnter script runs the autosave function -- it should be guarded by a condition so it doesn't happen more than once. Apparently someone left out the condition ... or it may have been intended as a sort of "safepoint"
> 
> regardless, it should _not_ happen when loading ...

I understand, thanks for the explanation. :)

**Comment by kevL** [comment]
| This should be the last one (gonna finish the game a few times again), Storm of Zehir next. :)

sounds ok :)

some of these can be fixed with a bit of time n effort

**Comment by Felipefpl** [comment]
> sounds ok :)
> 
> some of these can be fixed with a bit of time n effort

indeed, but it seems i spoke too soon :P - i remembered to report something i saw.

Safyia - when using energy drain on enemies i noticed a few things:

1 - there is no msg saying if i hit them or not.
2 - there is no msg saying how many levels were drained.
3 - there is no msg saying if the enemy has immunity against it.

Perhaps i may find something again, now i'm gonna bring vann to the party and let okku out (saved b4 the portal), let's see what happens.
